protobuf/mayastor.proto

```diff
@@ -247,6 +247,7 @@ message CreateNexusV2Request {
   uint64 resvKey = 6;    // NVMe reservation key for children
   uint64 preemptKey = 7; // NVMe preempt key for children
   repeated string children = 8; // uris to the targets we connect to
+  string nexusInfoKey = 9; // the key to use to persist the nexus info structure
```

Comment:
If it is added to the NexusV2, you need to add the key to here also https://github.com/openebs/mayastor-api/blob/master/protobuf/v1/nexus.proto#L52, in the v1 version of nexus proto

Reply:
We only want to support it for the V2 call (this is the call that the new control plane uses). The code changes to Mayastor will use the existing key if the "non-V2" call is made, so maintains backwards compatibility.